### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by SkochkoMan** [BoscoDomingo/SpotifyMatcher#6]
Hello! When running 1.1.2 from Powershell, I get the following error after processing just 86 songs:

```
Traceback (most recent call last):
  File "./main.py", line 163, in <module>
    result = sp.search(query_song_pair[0], limit=1)["tracks"]['items'][0]['id']
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./main.py", line 166, in <module>
    failed_matches_file.write(f"{query_song_pair[1]}\n")
  File "C:\Program Files\Python38\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2010' in position 3: character maps to <undefined>
```

**Comment by BoscoDomingo** [comment]
That's an issue with a Unicode character not being correctly mapped by internal Python libraries out of my control, which in turn causes the cascade of errors you see. Can you trace the song in question that caused the issue and remove it from the library you are trying to map to Spotify for now (simply search for it manually later), please? Let me know if that solves the issue!

**Comment by BoscoDomingo** [comment]
Should be fixed by #7
